<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="genreList">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>381</width>
      <height>361</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#deleteButton{
background-color: #ccdde8;
border: 1px white;
border-radius: 20px;
}
#deleteButton:hover{
color:rgb(73, 147, 221);
background-color: rgb(13, 27, 38);
border: 1px black;
border-radius: 20px;
}

</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="navigationBG" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#navigationBG {
background-color: #ccdde8;

}

</string>
    </property>
    <widget class="QLabel" name="systemLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>15</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#systemLabel {
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
color: #034488;
}</string>
     </property>
     <property name="text">
      <string>Movie
Ticketing
System</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/admin.png</normaloff>../img/admin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Crew</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/crew.png</normaloff>../img/crew.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/logout.png</normaloff>../img/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ticketsButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>209</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#ticketsButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#ticketsButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Bookings</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/tickets.png</normaloff>../img/tickets.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cinemaButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>250</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#cinemaButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#cinemaButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Cinema</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/cinema.png</normaloff>../img/cinema.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="genreButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>300</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#genreButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#genreButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Genres</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/genre.png</normaloff>../img/genre.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="movieButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#movieButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#movieButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Movies</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/movies.png</normaloff>../img/movies.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="showButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>130</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#showButton{
background-color: TRANSPARENT;
border: TRANSPARENT;
border-radius: 15px;
}

#showButton:hover{
background-color: rgb(236, 236, 236);
}</string>
     </property>
     <property name="text">
      <string>Shows</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../img/movies.png</normaloff>../img/movies.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
